main.js

```diff
@@ -1,51 +1,35 @@
-const fs = require('fs')
+const fs = require("fs");
 
-const All_employee = {
+const stringTo2dArray = (string, d1, d2) => string.split(d1).map(x => x.split(d2));
 
-};
-const All_user={
-    ID:"",
-    First_name:"",
-    Last_name:"",
-    email:"",
-    gender:"",
-    IP_adress:"",
-    color:"",
-    IP_parent:""
-}
-function stringTo2dArray(string, d1, d2) {
-    return string.split(d1).map(function(x){return x.split(d2)});
-}
-readPlain = () =>  {
-fs.readFile('MOCK_DATA.csv', (err, data) => {
+const readPlain = () => {
+  fs.readFile("MOCK_DATA.csv", (err, data) => {
     if (err) throw err;
-    let k=0;
-    let text = data.toString();
-    const myArray = stringTo2dArray(text, /\r?\n/ , ",");
-
-     for( let i=1;i<myArray.length-1;i++){
```

Comment:
Try to get benefit from javascript array utils, here I used map to generate new array without the need to declare **all_user** object